go/vt/vttablet/tabletserver/messager/message_manager.go

```diff
@@ -153,6 +160,18 @@ func newMessageManager(tsv TabletService, table *schema.Table, conns *connpool.P
 	return mm
 }
 
+// buildSelectColumnList is a convenience function that
+// builds a 'select' list for the user-defined columns.
+func buildSelectColumnList(t *schema.Table) string {
+	buf := new(bytes.Buffer)
+	prefix := ""
+	for _, c := range t.MessageInfo.Fields {
+		fmt.Fprintf(buf, "%s%s", prefix, c.Name)
```

Comment:
Do you need to quote the column name, to be safe?
Also, bytes.Buffer has WriteString() which might be simpler to use:
for i, c := range t.MessageInfo.Fields {
   if i > 0 {
     buf.WriteString(", ")
  }
  buf.WriteString(c.Name)
}